src/components/BackgroundVideo/BackgroundVideo.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { s } from "./";
+
+export default function BackgroundVideo(props){
+
+	const {
+		videoSrc,
+		posterSrc = {}
+	} = props.video;
+
+	return (
+		<div className={ s.wrapper }>	
+			<video poster={ posterSrc }>
```

Comment:
Before the `mp4`, if we include
```
<source
    src={ sources["webm"] }
    type="video/webm"
/>
```
Then we can save the user a hefty amount of filesize if they're using a browser thats supports webm! If it's not supported then it'll just fall-through and use the mp4 instead.